src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
+
+  for (const key in robot) {
+    if (newObj[robot[key]]) {
```

Comment:
What if the value turn out to be on of the falsy values, like 0 or null?)

```suggestion
    if (key in newObj) { // or use hasOwnProperty
```

Reply:
Agree. Value can be 0. So you rewrite this value but should return null. Use hasOwnProperty )